src/content/docs/start-here/getting-started.mdx

```diff
@@ -87,6 +89,12 @@ Configure a basic resource that ties into RBAC for permissions. Following is an
         ```bash
         git clone git@github.com:project-kessel/inventory-api.git
         cd inventory-api
+        # copy the schema.zed file we created earlier to deploy/schema.zed
+        cp ../schema.zed deploy/schema.zed
```

Comment:
I don't think you need to copy zed schema for Inventory, as long as you do it for Relations.